FarsiLibrary.Win/Controls/FADatePicker.cs

```diff
@@ -63,6 +64,16 @@ public FADatePicker()
             PopupShowing += OnInternalPopupShowing;
             Text = FALocalizeManager.Instance.GetLocalizerByCulture(mv.MonthViewControl.DefaultCulture).GetLocalizedString(StringID.Validation_NullText);
             FormatInfo = FormatInfoTypes.ShortDate;
+            
+            if (mv.MonthViewControl.DefaultCulture.Equals(mv.MonthViewControl.PersianCulture))
+                scroller = new Helpers.PersianCalendarScroller(this);
```

Comment:
The folder / namespace can be changed to reflect what it contains, e.g. `Scrollers` or something similar (rather than Helpers).

Reply:
the classes moved to Scrollers folder and the namespace changed to Scrollers